Mathlib/Analysis/Meromorphic/Order.lean

```diff
@@ -325,9 +363,50 @@ theorem order_ne_top_of_isPreconnected {y : 𝕜} (hU : IsPreconnected U) (h₁x
     (hf y hy).order ≠ ⊤ :=
   (hf.exists_order_ne_top_iff_forall ⟨nonempty_of_mem h₁x, hU⟩).1 (by use ⟨x, h₁x⟩) ⟨y, hy⟩
 
-/-- If the target is a complete space, then the set where a mermorphic function has zero or infinite
-order is discrete within its domain of meromorphicity. -/
-theorem codiscrete_setOf_order_eq_zero_or_top [CompleteSpace E] :
+theorem eventually_analyticAt {f : 𝕜 → E} {x : 𝕜}
+    (h : MeromorphicOn f U) (hx : x ∈ U) : ∀ᶠ y in 𝓝[U \ {x}] x, AnalyticAt 𝕜 f y := by
+  rcases h x hx with ⟨n, g, r, hgr⟩
+  have : U \ {x} ∩ EMetric.ball x r ∈ 𝓝[U \ {x}] x :=
+    inter_mem_nhdsWithin _ (EMetric.ball_mem_nhds _ hgr.r_pos)
+  filter_upwards [this] with y ⟨hy, h'y⟩
+  have hyx : y ≠ x := by simpa using hy.2
+  have : ContinuousAt f y := by
```

Comment:
I'm not sure if extracting this into a separate lemma would be useful: what do you think?